### PR TITLE
Add solana swap

### PR DESCRIPTION
**Pull request by nathanielparke**

### CODE REVIEW

**Comment by nathanielparke** [comment]
based on https://github.com/solana-labs/oyster-swap

File: src/components/swap/AddToLiquidity.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    }
+  };
+
+  return (
+    <div>
+      <Popover
+        trigger="hover"
+        content={
+          <div style={{ width: 300 }}>
+            Liquidity providers earn a 0.3% fee on all trades proportional to
```
Comment (askmike) [inline src/components/swap/AddToLiquidity.tsx:61]: 👀

**Comment by nathanielparke** [comment]
Closing for now, due to using https://github.com/project-serum/oyster-swap instead. There are possible plans in the works to merge these two repos, but will look significantly different than what this PR looks like.

**Comment by askmike** [comment]
for people who host a dex UI and strand here, for as long as these are seperate you need to host swap separate - see: https://serum-academy.com/en/serum-swap/ (and hook up the menu link yourself)

**Pull request closed without merging**